workspaces/adventure-pack/goodies/typescript/Number.prototype.chr/index.test.ts

```diff
@@ -0,0 +1,73 @@
+import { describe, expect, it } from "@jest/globals";
+
+import "./index";
+
+describe("Number.prototype.chr", () => {
+  describe("convert ASCII codepoint to ASCII string", () => {
+    it.each([
+      { codepoint: 65, expected: "A" },
+      { codepoint: 10, expected: "\n" },
+      { codepoint: 61, expected: "=" },
+      { codepoint: 126, expected: "~" },
+      { codepoint: 32, expected: " " },
+      { codepoint: 48, expected: "0" },
+    ])(
+      "convert ASCII codepoint $codepoint to ASCII string $expected",
```

Comment:
small nit: I think we should be consistent on the conjugation for the descriptions, I think a reason for the function name being "it" is so that it reads like "it does something"

Here that would be "it convert ASCII" therefore doing "it _converts_ ASCII" would be better.